res/skins/Shade/style.qss

```diff
@@ -83,34 +84,273 @@ WEffectSelector::checked
 	qproperty-layoutAlignment:AlignTop;
 }
 
-QTableView, QTextBrowser, QTreeView {
-  border: 1px solid #656565;
-  background: transparent; color: #ACACAC;
-  /* we use  "background: transparent" as workaround, else "alternate-background-color" wont work, QT 4.7.x bug?*/
-  alternate-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(43, 43, 43, 255), stop:0.049 rgba(43, 43, 43, 255), stop:0.050 rgba(43, 43, 43, 100), stop:0.949 rgba(43, 43, 43, 100), stop:0.95 rgba(43, 43, 43, 255), stop:1 rgba(43, 43, 43, 255));}
 
-QTableView:focus, QTreeView:focus {
-  border: 1px solid #FF6600;
-}
-/* checkbox in library "Played" column */
-QTableView::indicator { width: 12px; height: 12px;}
-QTableView::indicator:checked { background: url(skin:/style/style_checkbox_checked.png);}
-QTableView::indicator:unchecked { background: url(skin:/style/style_checkbox_unchecked.png);}
 
-/*  BPM lock icon in the library "BPM" column. */
-#LibraryBPMButton::indicator:checked { image: url(:/images/library/ic_library_checked.png); }
-#LibraryBPMButton::indicator:unchecked { image: url(:/images/library/ic_library_unchecked.png); }
+/*################################################################
+ ######  Library styling is hard  ###############################
+ ################### most of it was LateNight & Deere & Tango ##########
+############ added styling for AutoDJ & Recording etc ##### */
 
-/* Button in library "Preview" column */
-QPushButton#LibraryPreviewButton { width: 23px; height: 12px; background: transparent; border: 0; }
-QPushButton#LibraryPreviewButton:!checked{ image: url(skin:/style/style_library_preview_play.png); }
-QPushButton#LibraryPreviewButton:checked{ image: url(skin:/style/style_library_preview_pause.png); }
+QTableView,
+QTextBrowser,
+QTreeView {
+  border: 1px solid #585858;
+  font-weight: normal;
+  color: #9e9e9e;
+  background-color: #0f0f0f;
+  alternate-background-color: #1a1a1a;
+  selection-color: #e6e6e6;
+  selection-background-color: #666;
+}
 
-QPushButton#LibraryBPMButton {background: transparent; border: 0;}
-QPushButton#LibraryBPMButton::checked {image: url(:/images/library/ic_library_checked.png);}
-QPushButton#LibraryBPMButton::!checked {image: url(:/images/library/ic_library_unchecked.png);}
+QTreeView {
+  min-height: 90px;
+  min-width: 90px;
+  border-width: 1px 1px 1px 0px;
```

Comment:
this can be removed. It's overriding the def from line 97, and it makes the selected tree items overlap the tree border in Dark & Summer Sunset theme

Reply:
done